inventory_management_api/config.py

```diff
@@ -0,0 +1,8 @@
+def configure_app(app):
+    app.config["REDIS_HOST"] = "localhost"
+    app.config["REDIS_PORT"] = 6379
+    app.config["DB_HOST"] = "localhost"
+    app.config["DB_USER"] = "user"
+    app.config["DB_PASSWORD"] = "password"
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider using environment variables for credentials</b></div>
<div id="fix">

Consider using environment variables for sensitive information like database credentials instead of hardcoding them. This improves security and flexibility.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1,8 +1,10 @@
 +import os
 +
  def configure_app(app):
      app.config["REDIS_HOST"] = "localhost"
      app.config["REDIS_PORT"] = 6379
      app.config["DB_HOST"] = "localhost"
 -    app.config["DB_USER"] = "user"
 -    app.config["DB_PASSWORD"] = "password"
 +    app.config["DB_USER"] = os.environ.get("DB_USER", "default_user")
 +    app.config["DB_PASSWORD"] = os.environ.get("DB_PASSWORD", "default_password")
      app.config["DB_NAME_1"] = "inventory_db1"
      app.config["DB_NAME_2"] = "inventory_db2"
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/trulyfurqan/eshop-fastapi/pull/8#issuecomment-2472732780>#b8ff8d</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged